src/spanned.rs

```diff
@@ -31,7 +31,12 @@ macro_rules! span_with_attrs_lo_hi {
         if attrs.is_empty() {
             mk_sp($lo, $hi)
         } else {
-            mk_sp(attrs[0].span.lo(), $hi)
+            // this path is already gone at latest rust-ap-* 627.0.0
+            if attrs[0].item.path.to_string() == "warn_directory_ownership" {
```

Comment:
Wow, does this mean that the rustc parser is adding an attribute internally to the AST node? That is...surprising 😅

Reply:
Yes. But it will be deleted at the current version rust-ap-*. Maybe because it's not good for AST.